core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/service/ClusterProxyService.java

```diff
@@ -182,11 +175,11 @@ private String getAccountId(Stack stack) {
     }
 
     private List<CcmV2Config> ccmV2Configs(Stack stack) {
-        InstanceMetaData primaryGateway = stack.getPrimaryGatewayInstance();
-        return List.of(new CcmV2Config(stack.getCcmV2AgentCrn(),
-                String.format(CCMV2_BACKEND_ID_FORMAT, stack.getCcmV2AgentCrn(), primaryGateway.getInstanceId()),
-                primaryGateway.getPublicIpWrapper(),
-                ServiceFamilies.GATEWAY.getDefaultPort()));
+        return stack.getGatewayHostGroup().orElseThrow().getInstanceMetaDataSet().stream().map(instanceMetaData ->
```

Comment:
Can we use stack.getNotTerminatedGatewayInstanceMetadata() here as in getClusterServiceConfigsForGWs method?

Reply:
sure